src/colmap/estimators/fundamental_matrix.cc

```diff
@@ -110,35 +96,17 @@ void FundamentalMatrixSevenPointEstimator::Estimate(
               f1(8) * (f2(0) * f2(4) - f2(1) * f2(3));
   coeffs(3) = f2(0) * t3 - f2(1) * t4 + f2(2) * t5;
 
-  Eigen::VectorXd roots_real;
-  Eigen::VectorXd roots_imag;
-  if (!FindPolynomialRootsCompanionMatrix(coeffs, &roots_real, &roots_imag)) {
-    return;
-  }
-
-  models->reserve(roots_real.size());
-
-  for (Eigen::VectorXd::Index i = 0; i < roots_real.size(); ++i) {
-    const double kMaxRootImag = 1e-10;
-    if (std::abs(roots_imag(i)) > kMaxRootImag) {
-      continue;
-    }
-
-    const double lambda = roots_real(i);
-    const double mu = 1;
-
-    Eigen::MatrixXd F = lambda * f1 + mu * f2;
-
-    F.resize(3, 3);
-
-    const double kEps = 1e-10;
-    if (std::abs(F(2, 2)) < kEps) {
-      continue;
-    }
+  const double inv_c0 = 1.0 / coeffs(0);
```

Comment:
This can potentially result in inf? In the polynomial solver before there was a ``RemoveLeadingZero`` operation.

Reply:
Should be addressed now. Thanks.